lms/static/sass/base/_reset.scss

```diff
@@ -96,4 +96,5 @@ td { vertical-align: top; }
   @page { margin: 0.5cm; }
   p, h2, h3 { orphans: 3; widows: 3; }
   h2, h3 { page-break-after: avoid; }
+  .donotprint { display:none !important; }
```

Comment:
I am against this entirely. This could very easily be misused by course staff. If you really want stuff to not print in your course, you can inject that CSS directly. If we include this, we have two options:

1. We document this - thus implicitly endorsing hiding your content to those who may need to print for a11y reasons. I don't like this.
2. We don't document this - thus it's a secret that really only you know about. So, why not just ask you to inject this, really teeny amount of css, directly into your unit.